modules/apps/27-interchain-accounts/keeper/keeper.go

```diff
@@ -97,15 +97,15 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 }
 
 // GetPort returns the portID for the interchain accounts module. Used in ExportGenesis
-func (k Keeper) GetPort(ctx sdk.Context) string {
+func (k Keeper) GetPort(ctx sdk.Context, portID string) string {
```

Comment:
I don't think we need this function? Maybe `HasPort` instead?

Reply:
Correct, we don't need this! I left it in to satisfy the compiler and was planning on removing/adapting to `GetAllPorts` in a follow up for the genesis work.

Reply:
Seems a bit wasteful to store the portID under

`ports/{portID} -> {portID}`

You could just do:

`ports/{portID} -> []byte{0x01}`

And then retrieve the portID from the key if the value is non-nil. Just a nit

Reply:
Yep, agreed. The value is being stored a single byte `[]byte{0x01}` on line 108. 

`GetPort` is a little redundant at the moment as the only place it is used is for the genesis state, but for #448 we'll want to export all ports so I will update in a follow up with the genesis state work to something like:

```
func (k Keeper) GetAllPorts(ctx sdk.Context) []string {
	store := ctx.KVStore(k.storeKey)
	iterator := sdk.KVStorePrefixIterator(store, []byte(types.PortKeyPrefix))

	var ports []string
	for ; iterator.Valid(); iterator.Next() {
		keySplit := strings.Split(string(iterator.Key()), "/")

		ports = append(ports, keySplit[1])
	}

	return ports
}
```